assets/js/src/core/components/grid/grid.tsx

```diff
@@ -292,7 +292,7 @@ export const Grid = ({ enableMultipleRowSelection = false, modifiedCells = [], s
     </div>
   ), [table, modifiedCells, data, columns, rowSelection, internalSorting])
 
-  function getModifiedRow (rowIndex: string | number): GridProps['modifiedCells'] {
+  function getModifiedRow (rowIndex: string): GridProps['modifiedCells'] {
     return memoModifiedCells.filter(({ rowIndex: rIndex }) => String(rIndex) === rowIndex) ?? []
   }
```

Comment:
the incoming rowIndex ist typed to always be a string, while the rIndex has been a number in the cases I have seen. 
String() will convert it so its types are comparable in any case